src/api/ApiLinks.jsx

```diff
@@ -1 +1,4 @@
-export const CuisinesURL = `${process.env.REACT_APP_BACKEND_URL}/api/cuisines/getAllCuisines`;
\ No newline at end of file
+export const CuisinesURL = `${process.env.REACT_APP_BACKEND_URL}/api/cuisines/getAllCuisines`;
```

Comment:
@KatePolyakov can u merger main and change the folder accordingly 
please thank

Reply:
also Delete this ApiLinks file